client/src/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Axios from 'axios';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Probably did so in future pull request, but make sure to bind "this" to your getCat function